pages/services/edge-lb/1.2/installing/index.md

```diff
@@ -77,11 +77,7 @@ If you need to deploy a local Universe containing your own set of packages, you
     sudo make base
     ```
 
-3. Obtain the Edge-LB stub universe JSON files from the support downloads site. Note that there are two files required:
-- `stub-universe-edgelb.json`
-- `stub-universe-edgelb-pool.json`
-
-4. To add the JSON definitions to the universe, use the `add-stub-universe.sh` script.  Each run of the `add-stub-universe.sh` script will process the JSON file, generate the necessary JSON and Mustache files, and add them to `stub-repo/packages/<X>/<packagename>`.  
+3. Use `add-stub-universe.sh` script to add to the Universe the JSON definitions obtained in [Obtaining package artifacts](pages/services/edge-lb/1.2/installing/#obtaining-package-artifacts) section.  Each run of the `add-stub-universe.sh` script will process the JSON file, generate the necessary JSON and Mustache files, and add them to `stub-repo/packages/<X>/<packagename>`.
```

Comment:
We call sections `steps` sometimes, I think in this case:

to the Universe the JSON definitions obtained `during` [Obtain .. `step`.